src/api_v1.py

```diff
@@ -14,6 +14,94 @@
 from . import enums
 from . import gwtm_io
 
+@app.route('/health')
+def health_check():
+    return jsonify({"status": "ok"}), 200
+
+@app.route('/service-status')
+def service_status():
+    import os
+    
+    status = {
+        "database_status": "unknown",
+        "redis_status": "unknown",
+        "details": {
+            "database": {},
+            "redis": {}
+        }
+    }
+    
+    # Debug info
+    print("Checking database connection...")
+    
+    # Check database connection with detailed info
+    try:
+        # Get connection parameters from environment
+        db_host = os.environ.get('DB_HOST', 'localhost')
+        db_port = os.environ.get('DB_PORT', '5432')
+        db_user = os.environ.get('DB_USER', 'unknown')
+        db_name = os.environ.get('DB_NAME', 'unknown')
+        
+        # Store connection info
+        status["details"]["database"] = {
+            "host": db_host,
+            "port": db_port,
+            "name": db_name
+        }
+        
+        # Test actual connection
+        result = db.session.execute("SELECT 1").fetchone()
+        if result and result[0] == 1:
+            status["database_status"] = "connected"
+            print(f"Database connection successful: {db_host}:{db_port}/{db_name}")
```

Comment:
Eventually, this probably needs to be a logger over a print statement.

Reply:
Done